vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -690,6 +690,18 @@ func (l *terminationLoggingListener) Accept() (net.Conn, error) {
 		return nil, err
 	}
 
+	// ignore connections to local IP. Those clients better know what they are doing.
+	host, _, splitErr := net.SplitHostPort(c.LocalAddr().String())
+	if splitErr != nil {
+		return c, err
+	}
+	if ip := net.ParseIP(host); ip == nil {
+		return c, err
+	} else if ip.IsLoopback() {
+		klog.V(4).Infof("Accepted new local connection from %s very late in the graceful termination process (more than 80%% has passed).", c.RemoteAddr().String())
```

Comment:
what will happen with these connections? will they be processed?

thinking if we could set an appropriate header saying stop, retry in `X` seconds?

Reply:
they are legitimate connections and they are processed. But they show that the client does something risky.